config.yml

```diff
@@ -9,6 +9,13 @@ accounts:
       - 9000000000000000uelys
 build:
   main: cmd/elysd
+  ldflags:
+    - "-X github.com/cosmos/cosmos-sdk/version.Name=elys"
+    - "-X github.com/cosmos/cosmos-sdk/version.AppName=elys"
+    - "-X github.com/cosmos/cosmos-sdk/version.ServerName=elysd"
+    - "-X github.com/cosmos/cosmos-sdk/version.ClientName=elysd"
+    - "-X github.com/cosmos/cosmos-sdk/version.Version=v0.9.0"
+    - "-X github.com/cosmos/cosmos-sdk/version.Commit=b35a17d50b994831dcc9c2fa71bb0641af1d74a7"
```

Comment:
@kenta-elys I think you should get rid of those lines you probably used those for debugging earlier on

Reply:
@cosmic-vagabond Yep, no problem. I had rebase with main branch. I guess it was added by that command. Let me remove it. No problem.

Reply:
@cosmic-vagabond Done, removed.